### ISSUE TITLE
Where/how is progress stored?

### ISSUE THREAD

**Issue by P1xt** [jlord/git-it#160]
On first run of git-it, all steps are marked as "COMPLETED" (even though I've yet to do any of them). 

Could someone give me a hint as to where the progress should be getting stored so I can make sure I don't have some sort of weird permissions issue or something?

**Comment by jlord** [comment]
That's very strange. They should not be marked completed initially. I've seen git-it used across many user's systems now and haven't run into this. 

Just to be sure, you installed it fresh from `npm` and then ran it and when the menu loaded for the first time it said completed by each challenge? What operating system are you using and what version of Node/NPM (in case it ends up relevant)?

```
$ node -v
$ npm -v
```

**Comment by P1xt** [comment]
Ubuntu Linux 14.04 - 64 bit
Installed git-it fresh from npm today using "sudo npm install -g git-it"
$ node -v
v0.10.29
$ npm -v
1.4.14

Update: It's not just git-it isn't working on my box. I just installed the javascripting workshopper and that one showed completed across the board on first execution. I suspect there's supposed to be a config file somewhere that keeps track of everything, but for the life of me I can't figure out where said file is.

**Comment by jlord** [comment]
@P1xt here is the location where the workshops store their config data:

https://github.com/jlord/workshopper/blob/94a6b23068be8784cf04f76a60a36e71aceeec0b/workshopper.js#L48-L52

It is most likely a folder called `.config` in your home directory (`~`)

**Comment by P1xt** [comment]
Thanks much. The folder was right where you suggested ... and lo and behold, inside it there's a completed file filled with an array of every one of the modules. I emptied out the array and, bam, all fixed.

I've no idea how they all got marked complete, but I'm going to err on the side of "I probably ran the workshopper before and forgot I ran it" ... it doesn't feel like that's the case, but it's a perfectly plausible explanation.

Should anyone have this issue in the future, the fix was:
cd ~/.config/git-it

edit completed.json so that the contents is only
[]

edit current.json and delete it's contents entirely
